hawaii-logging/src/main/java/org/hawaiiframework/logging/http/DefaultHawaiiRequestResponseLogger.java

```diff
@@ -112,9 +112,9 @@ public void logResponse(final HttpServletRequest servletRequest,
         try (AutoCloseableKibanaLogField kibanaLogField = KibanaLogFields.logType(RESPONSE_BODY)) {
             LOGGER.info("Response '{}' is '{} {}' with content type '{}' and size of '{}' bytes.", request,
                     httpStatus.value(), httpStatus.getReasonPhrase(), contentType, contentLength);
-            if (contentTypeCanBeLogged(contentType)) {
+            if (contentTypeCanBeLogged(contentType) && contentLength != 0) {
```

Comment:
wouldn't we want to log the response headers even if the content length is zero?

Reply:
Yes thats right. Headers can still be useful even if there is no content,, I removed this from the if statement